plugins/nodebb-plugin-caiz/languages/en-US/caiz.json

```
@@ -45,5 +90,82 @@
+  "promote-to-manager": "Promote to Manager",
+  "ban-user": "Ban User",
+  "member": "Member",
+  "owner": "Owner",
+  "manager": "Manager",
+  "banned": "Banned",
+  "change-role": "Change Role",
+  "remove": "Remove",
+  "change-role-confirm": "Are you sure you want to change {username}'s role to {role}?",
+  "remove-member-confirm": "Are you sure you want to remove {username} from this community? This action cannot be undone.",
+  "remove-member-title": "Remove Member",
+  "categories": {
```

Biome (2.1.2)

[error] The key categories was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)